spring-pulsar-sample-apps/sample-pulsar-binder/src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+spring:
+  cloud:
+    function:
+      definition: supplier;consumer
+    stream:
+      bindings:
+        consumer-in-0:
+          destination: supplier-out-0
+          consumer:
+            use-native-decoding: true
+      pulsar:
+        bindings:
+          consumer-in-0:
+            consumer:
+              subscription-name: my-scst-sub1
+              schema-type: STRING
```

Comment:
I see you did not need to set schema type on producer. Is that because the payload is always `Object` and then it defaults to `Schema.BYTES` in that case?